### ISSUE TITLE
Current time in media launched

### ISSUE THREAD

**Issue by TyrkoDev** [alxhotel/chromecast-api#42]
I didn't find anything in the documentation about this topic, Is there any way to have the current time of the media launched ? For instance, if I add subtitles while watching a video, I don't want to restart the video from the beginning but where I was. If it doesn't exist, it could be a great feature !

**Comment by alxhotel** [comment]
Hi @TyrkoDev ,

Thanks for the suggestion !

To get the current time of the media you can use the `getStatus` and `status.currentTime` like this:

```js
device.getStatus(function (err, status) {
  if (err) return console.error(err)
  console.log('Current time: ', status.currentTime)
})
```

Note that `currentTime` will return the time in seconds (even with some decimal places).

For example the above example could return this:

```sh
Current time: 1796.663
```

PS: The above snippet might even be a full implementation of the feature `getCurrentTime`, so I will leave this feature in the roadmap :)

**Comment by alxhotel** [comment]
New method `getCurrentTime` released in 9112650f9e26686204d65ae329f8343639f79417
